crawl-ref/source/target.cc

```diff
@@ -2024,6 +2024,18 @@ targeter_bog::targeter_bog(const actor *a, int pow)
         affected_positions.insert(c);
 }
 
+targeter_gastronomic::targeter_gastronomic(const actor *a, int pow)
+    : targeter_smite(a, 1, 1, 1, {})
+{
+    
+}
+
+bool targeter_gastronomic::valid_aim(coord_def loc)
+{
+    coord_def delta = loc - you.pos();
```

Comment:
I would try allow aiming anywhere in the 90 degree cone and use a method similar to targeter_cone to preview where the expansion is going to go? Maybe using AFF_YES vs AFF_MAYBE to show the difference between short term and long term expansion